Assets/Scripts/Player/AimMode.cs

```diff
@@ -144,16 +147,33 @@ public void ThrowObject()
 
     public void changeCameraToPossession()
     {
-        ExitAimMode();
         if (_possessionController.CurrentPossession == null)
         {
+            ExitAimMode();
             return;
         }
         Transform pos = _possessionController.CurrentPossession.GetComponent<ClutterCamera>().CinemachineCameraTarget.transform;
```

Comment:
Also please change the name pos to something more descriptive. pos is used for position.